metric_providers/network/connections/proxy/container/provider.py

```diff
@@ -44,11 +44,6 @@ def check_system(self):
 
         raise MetricProviderConfigurationError('Tinyproxy needs to be version 1.11 or greater.')
 
-    def has_started(self):
-        return self._has_started
-
-    def get_stderr(self):
-        return self._ps.stderr.read()
 
     def get_docker_params(self):
 
```

Comment:
Can you please also refactor the class to use the base.py start_profiling method instead of overloading it.

Supplying extra switches is possible through the `self._extra_switches` variable